Lib/crypt.py

```diff
@@ -19,15 +19,20 @@ def __repr__(self):
         return '<crypt.METHOD_{}>'.format(self.name)
 
 
-def mksalt(method=None):
+def mksalt(method=None, *, log_rounds=12):
     """Generate a salt for the specified method.
 
     If not specified, the strongest available method will be used.
 
     """
     if method is None:
         method = methods[0]
-    s = '${}$'.format(method.ident) if method.ident else ''
+    if not method.ident:
+        s = ''
+    elif method.ident[0] == '2':
+        s = f'${method.ident}${log_rounds:02d}$'
```

Comment:
What if `log_rounds` is more than 99?

Reply:
`log_rounds` should be between 4 and 31. `crypt()` will return None if it is invalid or out of range. Should we silently bound `log_rounds` into the range 4 to 31, or raise an error, or keep all as is and let `crypt()` detect an error?

Reply:
I prefer to have a thin wrapper to the C crypt() function. Maybe add unit tests for invalid round values? Negative, 0, 999, etc.?